config/install.yaml

```diff
@@ -1411,7 +1451,7 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-        image: quay.io/clastix/capsule:v0.1.0
+        image: quay.io/clastix/capsule:v0.1.0-rc5
```

Comment:
Please, perform a `git fetch --all` on your local repository to ensure downloading all the git tags since this is going to be updated by the makefile using the latest git tag.

Reply:
done